packages/website/src/docs/button.mdx

```diff
@@ -4,34 +4,43 @@ component: Button
 description: Useful to communicate actions to your users.
 ---
 
-```js live
-<Button>New project</Button>
-```
+import { Button } from '@modulz/radix';
```

Comment:
I don't know why, but the build would fail without an explicit import of the component.